package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "arcads",
-  "version": "6.1.2",
+  "version": "6.1.3",
```

Comment:
@rachelpost, @jakma : This may be closer to a feature add than a path. Wondering if this should be `6.2.0` instead of `6.1.3`.